addok/http/base.py

```diff
@@ -139,8 +139,11 @@ def on_get(self, req, resp, **kwargs):
         if lon and lat:
             center = (lon, lat)
         filters = self.match_filters(req)
-        results = search(query, limit=limit, autocomplete=autocomplete,
-                         lat=lat, lon=lon, **filters)
+        try:
+            results = search(query, limit=limit, autocomplete=autocomplete,
+                             lat=lat, lon=lon, **filters)
+        except ValueError as e:
+            raise falcon.HTTPBadRequest(str(e))
```

Comment:
Mmmh I would raise a custom error AND add a catch all with a ValueError for generic errors. 413 looks to be exactly what we want here.